model/riscv_pmp_control.sail

```diff
@@ -218,7 +270,9 @@ function pmpCheck forall 'n, 'n > 0. (addr: xlenbits, width: atom('n), acc: Acce
     PMP_Success  => true,
     PMP_Fail     => false,
     PMP_Continue => match priv {
-                      Machine => true,
+                      Machine => if mseccfg.MMWP() == 0b1   /*If mseccfg.MMWP is set then default pmp policy for matching region not found is denied*/
```

Comment:
Why not move this check into MMLcheck given this is the only place the function is called?

Reply:
Yes, you are right, the function is removed and the check is moved where the function was called